include/drivers/gpio.h

```diff
@@ -112,6 +112,12 @@ extern "C" {
  */
 #define GPIO_INT_LEVELS_LOGICAL        (1U << 16)
 
+/** @cond INTERNAL_HIDDEN */
+#define GPIO_INT_ALL_MASK (GPIO_INT_LOW_0 | GPIO_INT_HIGH_1 | \
```

Comment:
Not required by the new implementation of `gpio_pin_interrupt_configure` function.